pkgs/common-updater/unstable-updater.nix

```diff
@@ -8,6 +9,7 @@
 # This is an updater for unstable packages that should always use the latest
 # commit.
 { url ? null # The git url, if empty it will be set to src.url
+, branch ? null
```

Comment:
```suggestion
, branch ? "master"
```

Reply:
More and more repos choose a different default branch name.

Reply:
Yes, but this is the git default. The intention is to simplify the code mostly, not to enforce some notion of a default branch. Setting the default value here means we always take the same code path.

Reply:
No, it isn't. The default branch is determined by the remote server. If the default branch on GitHub is develop it will work with the current code.

Reply:
Oh yeah, that's true.